Source/WebCore/css/CSSProperties.json

```diff
@@ -10785,14 +10770,19 @@
                 "url": "https://www.w3.org/TR/css-writing-modes-3/#propdef-direction"
             }
         },
-        "-webkit-text-decoration": {
+        "text-decoration": {
             "codegen-properties": {
                 "longhands": [
                     "text-decoration-line",
                     "text-decoration-style",
-                    "text-decoration-color"
+                    "text-decoration-color",
+                    "text-decoration-thickness"
```

Comment:
The order of these matter, and in the spec, it is listed as:

`<'text-decoration-line'> || <'text-decoration-thickness'> || <'text-decoration-style'> || <'text-decoration-color'>`

Please also remember to update the `parser-grammar-unused` field as well as the specification URL to point to https://www.w3.org/TR/css-text-decor-4/#propdef-text-decoration.